google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminClientImplTest.java

```diff
@@ -294,21 +314,26 @@ public void testCreateInstanceWithProcessingUnits() throws Exception {
   }
 
   @Test
-  public void testCreateInstanceWithBothNodeCountAndProcessingUnits() throws Exception {
```

Comment:
Are we renaming or removing this test `testCreateInstanceWithBothNodeCountAndProcessingUnits` ? If yes, why?

Reply:
This test was disabled since we removed the check in google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceAdminClientImpl.java: ``` Preconditions.checkArgument(
        instance.getNodeCount() == 0 || instance.getProcessingUnits() == 0,
        "Only one of nodeCount and processingUnits can be set when creating a new instance"); ```

Per earlier conversation, we decided to remove the check to reduce the hand-written layer.

Reply:
Ah got it.